project-base/src/Shopsys/ShopBundle/Resources/styles/styleguide/plugins/codemirror/theme/hopscotch.css

```diff
@@ -0,0 +1,34 @@
+/*
+
+    Name:       Hopscotch
+    Author:     Jan T. Sott
+
+    CodeMirror template by Jan T. Sott (https://github.com/idleberg/base16-codemirror)
```

Comment:
I cannot see license there - can we use it?

Reply:
This is under codemirror which is MIT - https://github.com/codemirror/CodeMirror/blob/master/LICENSE
https://github.com/codemirror/CodeMirror/blob/master/theme/hopscotch.css